src/tokens/$metadata.json

```diff
@@ -7,6 +7,8 @@
     "color/modern",
     "layout/bb-productive",
     "layout/bb-compact",
-    "layout/modern"
+    "layout/modern",
+    "bb-prod-sm-min",
```

Comment:
It feels like the `bb-prod-sm-min` and `modern-sm-min` files should be in the `layout` folder.

Reply:
Moved.